src/azure-cli/azure/cli/command_modules/privatedns/custom.py

```diff
@@ -940,9 +938,20 @@ def _privatedns_add_save_record(cmd, record, record_type, relative_record_set_na
         record_set = _convert_to_snake_case(record_set)
     except HttpResponseError:
         record_set = {"ttl": 3600}
+        is_empty = True
 
     _privatedns_add_record(record_set, record, record_type, is_list)
```

Comment:
It seems the record_set will not be defined when _record_show throw HttpResponseError